templates/zerver/register.html

```diff
@@ -100,7 +100,7 @@
             <label for="id_team_name" class="inline-block label-title">{{ _('Organization name') }}</label>
 
             <div class="help-box margin-top">
-                {{ _('This can be changed later on the settings page.') }}
+                {{ _('This can always be changed later. Keep it short!') }}
```

Comment:
Yeah, @rishig and I had a discussion about this yesterday. Agreed that `short name` and `organization name` aren't too distinct as labels, though I think the note is still helpful. Also - planning on adding some logic so that the input value of the short name field is forced / automatically corrected to only lowercase letters, numbers, and dashes